test-cases/longevity/longevity-multi-dc-rack-aware-with-znode-in-diff_dc.yaml

```diff
@@ -0,0 +1,25 @@
+test_duration: 800
+
+prepare_write_cmd:  ["cassandra-stress write cl=LOCAL_QUORUM n=20971520 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3,eu-northscylla_node_north=0) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=80 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5",
+                    ]
+
+stress_cmd: ["cassandra-stress write cl=LOCAL_QUORUM duration=720m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3,eu-northscylla_node_north=0) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=40 -pop 'dist=uniform(1..20971520)' -col 'n=FIXED(10) size=FIXED(512)' -log interval=5 -errors retries=50",
+             "cassandra-stress read  cl=LOCAL_QUORUM duration=720m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3,eu-northscylla_node_north=0) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=40 -pop 'dist=uniform(1..20971520)' -col 'n=FIXED(10) size=FIXED(512)' -log interval=5 -errors retries=50",
+             ]
+
+n_db_nodes: '3 3 0'
+n_loaders: '1 1'
+n_monitor_nodes: 1
+n_db_zero_token_nodes: '0 1 1'
+
+instance_type_db: 'i4i.4xlarge'
+zero_token_instance_type_db: 'i4i.xlarge'
```

Comment:
i4i.large won't be enough?

Reply:
Switched to i4i.large. 
Also open issue https://github.com/scylladb/scylladb/issues/21321 to get answer regarding instance with ebs-only storages. 
i checked, we already can configure additional ebs storage, but for now this config will affect all nodes. This could be fixed in another PR, so we can configure data_ebs_storage separately for data and zero nodes (at least for aws via provisioner)

Reply:
@patjed41 made a good point on the issue. Let's just verify this experimentally.